### ISSUE TITLE
문의 드립니다.

### ISSUE THREAD

**Issue by SY0326JH** [S-jooyoung/S-jooyoung.github.io#4]
node js는 처음 사용해보는 개발자입니다.
dependencies 계속 환경을 맞추는데 하나씩 에러가나서요

혹시 node 어떠한 버전을 사용하셨나요?

**Comment by S-jooyoung** [comment]
네 안녕하세요 :) 노드 버전은 v16.15.0을 사용하고 있습니다. 버전 맞추어 진행해 보시고 문제 있을 시 해당 에러 내용 첨부하여 이슈 남겨 주시면 확인해 보도록 하겠습니다!
